modules/aws-outposts-prereq-storage-csi.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-prereq-storage-csi_{context}"]
+= Preparing the CSI driver to support AWS Outposts
+
+You must update the Container Storage Interface (CSI) driver default storage class to be compatible with AWS Outposts.
```

Comment:
Which CSI driver are you refering to? AWS has Elastic Block Store (EBS) and Elastic File Service (EFS). Looks like EBS from what you say later in this doc, so you may want to call this out here.